src/science/landuse/landuse_constant.F90

```diff
@@ -1,18 +1,27 @@
 MODULE landuse_constant
+  ! this contains landuse-specific constants

```

Comment:
Don't you want this to be picked up by the documentation as well to give a quick purpose of the module